runtime/src/bank.rs

```diff
@@ -1600,6 +1600,20 @@ impl Bank {
             "Stakes cache is inconsistent with accounts-db. This can indicate \
             a corrupted snapshot or bugs in cached accounts or accounts-db.",
         );
+
+        // Ensure EpochStakes from the snapshot does not contain entries for invalid epochs.
+        // Since epoch stakes are computed for `epoch + 1`, EpochStakes therefor can have
+        // entries for epochs at or below `epoch + 1`.
```

Comment:
We should also ensure that it definitely *does* contain entries for valid epochs (ie epoch + 1)

Reply:
can we assert that on brand new cluster?

Reply:
Yeah if there's a snapshot it means we serialized at least one bank and it should always include epoch stakes right?